packages/types-algosdk/index.d.ts

```diff
@@ -341,33 +341,33 @@ export function decodeUnsignedTransaction(b: Uint8Array): Transaction;
 export function decodeSignedTransaction(b: Uint8Array): SignedTransaction;
 
 export function makeApplicationClearStateTxn(from: string, suggestedParams: any, appIndex: number,
-  appArgs?: Uint8Array[], accounts?: any, foreignApps?: any, foreignAssets?: any,
+  appArgs?: Uint8Array[], accounts?: string[], foreignApps?: any, foreignAssets?: any,
```

Comment:
Let's update types of `suggestedParams`, `foreignApps`, `foreignAssets` as well.

Reply:
yes